include/openmc/photon.h

```diff
@@ -99,9 +98,21 @@ class PhotonInteraction {
   // Bremsstrahlung scaled DCS
   xt::xtensor<double, 2> dcs_;
 
+  // Constant data
+  static constexpr int MAX_STACK_SIZE =
+    7; //!< maximum possible size of atomic relaxation stack
```

Comment:
It looks like something is getting messed up on CI now after you made the change with fmtlib. I think this has to do with this static constexpr data member and the fact that we are compiling with `-std=c++14`. It may need to be redeclared at namespace scope (according to [this](https://en.cppreference.com/w/cpp/language/static), see clause at bottom of "Constant static members" section)?